googlemaps/client.py

```diff
@@ -133,30 +133,42 @@ def __init__(self, key=None, client_id=None, client_secret=None,
         self.sent_times = collections.deque("", queries_per_second)
 
     def _get(self, url, params, first_request_time=None, retry_counter=0,
-             base_url=_DEFAULT_BASE_URL, accepts_clientid=True, extract_body=None):
+             base_url=_DEFAULT_BASE_URL, accepts_clientid=True,
+             extract_body=None, requests_kwargs=None):
```

Comment:
That was very similar! The previous change exposed the requests kwargs at the client level for settings that should be applied to every request, eg proxy auth. This new changes exposes the same args internally per request, so that we can do things like set the `stream` setting, which we need for photo downloads.